Readme.md

```diff
@@ -203,8 +203,8 @@ that you can start or bind or map or whatever you want to do. To handle errors,
 for instance, we could do the following:
 
 ```swift
-provider = ReactiveSwiftMoyaProvider<GitHub>()
-provider.request(.userProfile("ashfurrow")).start { event in
+provider = MoyaProvider<GitHub>()
```

Comment:
Curious how many people will "forget" this and end up with unexpected results. :/